ReactVR/js/Views/Prefetch.js

```diff
@@ -52,6 +52,37 @@ export default class RCTPrefetch extends RCTBaseView {
       return;
     }
 
+    const onError = () => {
+      onLoadEnd();
+    };
+
+    const onLoad = texture => {
+      // call onLoad in React
+      if (texture !== undefined) {
+        RCTPrefetch.addToCache(uri, texture);
+        console.log('ONLOAD', uri, texture)
```

Comment:
May want to remove the console.log from this. 

_* Not a maintainer, was just curious about this PR